base_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
```

Comment:
Just as required for this milestone, kindly ensure a nameable object is assigned in the constructor from params to an instance variable.

```suggestion
require './nameable'
`attr_accessor :nameable`
```

<img width="559" alt="image" src="https://github.com/jaimevillegas/oop-school-library/assets/64643995/910db0eb-b5a8-4900-95e9-43442ff1f937">